local_planner/cfg/local_planner_node.cfg

```diff
@@ -32,7 +31,6 @@ gen.add("stop_in_front_", bool_t, 0, "Enable stop in front of the obstacle", Fal
 gen.add("use_back_off_", bool_t, 0, "Enable functionality to move backwards if an obstacle is too close", False)
 gen.add("use_VFH_star_", bool_t, 0, "Build lookahead-tree", True)
```

Comment:
I've been meaning to get rid of these two as well. We pretty much always use VFH-star these days, and if someone wants to really only plan one node, they could set `n_expanded_nodes` to 1, which should have the same effect as setting `use_VFH_star` to `False`.
The back-off has also been disabled by default for a while, do you have an opinion on that? Can we get rid of it?

Reply:
yes, I think so. @baumanta @jkflying your opinions?

Reply:
Agreed.

Reply:
let's do it in a separate PR